Makefile

```
@@ -2,6 +2,7 @@ PWD=$(shell pwd)
 NODE_IMAGE=node:20.12.2
 IMAGE_NAME=oca-frontend
 CONTAINER_NAME=oca-frontend-app
+PORT=3000
```

Define the container port and the host port 

```suggestion
HOST_PORT=80
CONTAINER_PORT=3000
```